data/vocabularyList.js

```diff
@@ -1,196 +1,272 @@
+import dayjs from "dayjs";
```

Comment:
i think dayjs is not required anymore, if you're not using it anywhere else, you can deinstall it fully.